src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.replace('\n', '').split(';');
+
+  for (let line of lines) {
+    line = line.trim();
+
+    if (!line.includes(':')) {
+      continue;
+    }
+
+    const [property, value] = line.split(':');
+
+    const cleanProperty = property.trim();
+    const cleanValue = value.replace(';', '').trim();
+
+    styles[cleanProperty] = cleanValue;
+  }
+
+  return styles;
```

Comment:
I think u can do it using the reduce method, I recommend refactoring it into a task using it. But up to u

```suggestion
  return sourceString.split(';').reduce((style, line) => {
    const [property, value] = line.trim().split(':');

    ............

    return style;
  }, {});
```